docs/bits/attribute/L_bijective.md

```diff
@@ -0,0 +1,7 @@
+**bijective attribute**
+
+When used with a function domain will restrict solutions to functions that are both [injective](https://github/conjure-cp/conjure/docs/bits/attribute/L_injective.md) and [surjective.](https://github/conjure-cp/conjure/docs/bits/attribute/L_surjective.md)
+
+This means that every element of the defined set has a unique mapping to a range element and every range element has a defined element that maps to it.
+
+See it demonstrated [here.](https://github/conjure-cp/conjure/docs/notebooks/functionDemonstration.ipynb)
```

Comment:
very tiny nitpick: `[here](URL).` is better style.

Reply:
corrected this too